django/contrib/staticfiles/handlers.py

```diff
@@ -1,6 +1,7 @@
 from django.conf import settings
 from django.contrib.staticfiles import utils
 from django.contrib.staticfiles.views import serve
+from django.core import urlresolvers
```

Comment:
I meant, what is the purpose of the changes? They don't appear to be part of the original patch. Are they related to supporting unicode in `SlugField`?